src/node.cc

```diff
@@ -4493,6 +4493,7 @@ int Start(int argc, char** argv) {
   Init(&argc, const_cast<const char**>(argv), &exec_argc, &exec_argv);
 
 #if HAVE_OPENSSL
+  crypto::UseExtraCaCerts(secure_getenv("NODE_EXTRA_CA_CERTS"));
```

Comment:
This will happen before OpenSSL's initialization. I'd rather do it in the `node_crypto.cc`'s `Initialize`. Cons: error messages are not loaded yet at this point.